intro-HPC/title.tex

```diff
@@ -72,7 +72,7 @@
 \strong{\underbar{Contents:}}
 
 This \strong{Beginners Part} of this tutorial gives answers to the typical
-questions that a new \hpc user has. The aim is to learn how to use of the
+questions that a new \hpc user has. The aim is to learn how to make use of the
```

Comment:
using it is not the hard part, getting useful results out of it is what we're aiming for here.